objcgen/objcgenerator.cs

```diff
@@ -192,20 +192,17 @@ protected override void Generate (Type t)
 			var default_init = false;
 			List<ConstructorInfo> constructors;
 			if (ctors.TryGetValue (t, out constructors)) {
+				// First get the unavailable init ctor selectors in parent class
+				var unavailableInits = GetUnavailableParentCtors (t, constructors);
```

Comment:
That needs a comment, just to have a recurring _why_ question, with a link to our documentation (for more details).